corehq/apps/app_manager/models/common.py

```diff
@@ -2186,6 +2143,8 @@ def uses_usercase(self):
 
 
 class AdvancedForm(IndexedFormBase, NavMenuItemMediaMixin):
+    from corehq.apps.app_manager.models.schedules import FormSchedule
```

Comment:
Yeah, I don't love the circular imports. I don't have a great sense of how many there will be if I get all of the app manager models into smaller files, but there will definitely be sone. I'm hoping that the tradeoff of having more manageable files is worth it.